test/test-color-scheme.rb

```diff
@@ -78,5 +80,35 @@ def test_default
       assert_equal(expected_schema_keys.sort,
                    Test::Unit::ColorScheme.default.to_hash.keys.sort)
     end
+
+    def test_available_colors
```

Comment:
`TestFor8Colors` is a test case for 8 colors case.
Here is not suitable location. 
Newly test case such as `TestGuess` or something is better.

Reply:
Oh, I was skimming quickly and hadn't noticed it was in a namespaced class. Oops. Will fix.